gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -65,7 +65,41 @@ paths:
             The user task with the given key cannot be completed.
             More details are provided in the response body.
           $ref: "#/components/responses/ProblemResponse"
-
+  /user-tasks/{userTaskKey}/assignment:
+    post:
+      summary: Assigns a user task.
+      description: Assigns a user task with the given key to the given assignee.
+      parameters:
+        - name: userTaskKey
+          in: path
+          required: true
+          description: The key of the user task to complete.
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UserTaskAssignmentRequest"
+      responses:
+        '202':
```

Comment:
🔧 We don't issue 202s yet. We'll do this when we implement task listeners.

Reply:
Removed, thank you.